include/ui/input-text-model.hpp

```diff
@@ -25,7 +25,9 @@
 
 #include "core/memory-access.hpp"
 
-using length_t = unsigned ;
+namespace InputText{
+    using length_t = unsigned ;
```

Comment:
clang format. Any reason this is inside a namespace and not just the class itself?